federation/manifests/federation-apiserver-deployment.yaml

```diff
@@ -23,7 +23,6 @@ spec:
         - --service-cluster-ip-range={{.FEDERATION_SERVICE_CIDR}}
         - --secure-port=443
         - --advertise-address={{.FEDERATION_API_HOST}}
-        - --admission-control=NamespaceLifecycle,LimitRanger,SecurityContextDeny,ServiceAccount,ResourceQuota
```

Comment:
We are not supporting these admission controls in 1.3. Ref https://github.com/kubernetes/kubernetes/pull/26298#issuecomment-221697124